packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -266,6 +266,10 @@ abstract contract Sphinx {
         return sphinxUtils.getGnosisSafeProxyAddress(sphinxConfig);
     }
 
+    function sphinxVersion() public pure returns (string) {
+        return sphinxUtils.getSphinxVersion();
+    }
```

Comment:
Will be used by the CLI to detect the installed version of the Sphinx library contracts.